app/src/components/DelayActions.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { useTheme } from '@aragon/ui'
+
+import DelayButton from './DelayButton'
+import ResumeIcon from '../assets/resume.svg'
+import PauseIcon from '../assets/pause.svg'
+
+import STATUS from '../delay-status-types'
+
+const getMainActionProps = (status, theme) => {
+  switch (status) {
+    case STATUS.ONGOING:
+      return {
+        mode: 'normal',
```

Comment:
just for consistency, i would start the strings all with uppercase or lowercase

Reply:
The thing is that some of the values are Components which by convention start with Upper case, and the labels are needed to start with uppercase as well.